models/ssrc_fog_x/ssrc_fog_x.sdf.jinja

```diff
@@ -911,7 +912,7 @@
     <!--} END CAMERA -->
 
     <!-- RP LIDAR {-->
-    <link name="rplidar_link">
+    <link name="2d_scanner_link">
```

Comment:
I think if we rename the lidar link we need to modify also the regex string to make mavlink plugin to find correct link.
https://github.com/tiiuae/PX4-SITL_gazebo/blob/7dcc3ee06d6dbf4f30f6bdb89a2bbae84ebc210f/include/gazebo_mavlink_interface.h#L76-L80